base/applications/regedit/clb/clb.c

```diff
@@ -280,10 +246,8 @@ DllMain(IN HINSTANCE hinstDLL,
         ClbWndClass.lpfnWndProc = ClbWndProc;
         ClbWndClass.cbClsExtra = 0;
         ClbWndClass.cbWndExtra = sizeof(PCLB_PRIVATEDATA);
-        ClbWndClass.hInstance = hinstDLL,
-                    ClbWndClass.hIcon = NULL;
-        ClbWndClass.hCursor = LoadCursor(NULL,
-                                         (LPWSTR)IDC_ARROW);
+        ClbWndClass.hInstance = hinstDLL, ClbWndClass.hIcon = NULL;
```

Comment:
This looks strange to me. Should there be a semicolon in between instead of the comma? Or is/was this correct the way it is? @HBelusca

Reply:
It's the same thing in this case but I get in trouble if I code like that.

Reply:
Impressive. I was a bit surprised actually that the old code did even compile without a warning. I changed it meanwhile to the semicolon.